types/react/test/elementAttributes.tsx

```diff
@@ -171,6 +171,12 @@ const testCases = [
             <custom-element exportparts="nested" />
         </template>
     </>,
+    <link rel="expect" href="#lead-content" blocking="render" />,
+    <link rel="expect" href="#lead-content" blocking="render render" />,
```

Comment:
This is technically invalid since the tokens aren't unique. It's only here for future compat since the blocking attribute takes a space separated list of blocking tokens. And we currently can't express unique tokens in TypeScript anyway.